instrumentation/runtime/runtime.go

```diff
@@ -26,18 +26,42 @@ import (
 
 // Runtime reports the work-in-progress conventional runtime metrics specified by OpenTelemetry
 type runtime struct {
-	meter    metric.Meter
-	interval time.Duration
+	meter              metric.Meter
+	minGCStatsInterval time.Duration
+}
+
+// Option supports configuring optional settings for runtime metrics.
+type Option func(*runtime)
```

Comment:
Should we standardize the `Option/Apply` interface approach in `opentelemetry-go-contrib` the same as in `opentelemetry-go`, i.e. the approach [described here](https://github.com/open-telemetry/opentelemetry-go/issues/536)?

Reply:
Fixed.